docs/examples/nodejs-read-volatile-layer.md

```diff
@@ -0,0 +1,176 @@
+# Example of how to read the layer data on Node.js using @here/olp-edge-sdk-dataservice-read and @here/olp-edge-sdk-authentification
+
+This example shows how to retrieve partition metadata and partition data using the OLP SDK for TypeScript.
+
+## Build and run the app on Node.js
+
+Requirements:
```

Comment:
You already have the dependencies listed in the root README.md so i dont think you need to mantain another list. Maybe just a "see README.md#dependencies for dependencies" or similar.

Reply:
Fixed